pkgs/applications/networking/browsers/w3m/default.nix

```diff
@@ -1,16 +1,18 @@
 { stdenv, fetchurl
-, sslSupport ? true
-, graphicsSupport ? false
-, mouseSupport ? false
-, ncurses, openssl ? null, boehmgc, gettext, zlib
-, imlib2 ? null, xlibsWrapper ? null, fbcon ? null
-, gpm-ncurses ? null
+, ncurses, boehmgc, gettext, zlib
+, sslSupport ? true, openssl ? null
+, graphicsSupport ? true, imlib2 ? null
+, x11Support ? graphicsSupport, libX11 ? null
```

Comment:
Okay then, rebuilding the thing right now (will take a while with `gcc` update).

I decided to do this:

   w3m = callPackage ../applications/networking/browsers/w3m { };

   # Should always be the version with the most features
   w3m-full = w3m;
 
   # Version without X11
   w3m-nox = w3m.override {
     x11Support = false;
   };
 
   # Version for batch text processing, not a good browser
   w3m-batch = w3m.override {
     graphicsSupport = false;
     x11Support = false;
     mouseSupport = false;
   };

Does this look good to you?

Thank you, too!